### ISSUE TITLE
Epoch 0.3.2.0 / A3EAI 0.8.4d - AI soldier kill action causes red arrow to appear over corpse

### ISSUE THREAD

**Issue by moosemilker** [dayzai/A3EAI#10]
Repro:
1. With latest A3EAI 0.8.4d code, kill any A3EAI soldier unit

Result: 
Dead AI corpse has giant red arrow pointing to corpse

Expected: 
 - No red marker, or option to turn it off in config (debug marker parameter) 
 - Gut feeling is it should be off by default

More info:
A3EAI implemented at any of the servers at: http://moosemilker.com - click to join map of your choice.

Thanks so much for your time.

PS - Thank you for already fixing the AI units spawning without uniforms.

**Comment by donkeypunchepoch** [comment]
That is the krypto finder, but I believe is only suppose to appear when you as a player are close to the krypto dropped.

**Comment by moosemilker** [comment]
Oh!  Gotcha.  I turned the 'krypto finder' option on when I 1st saw it as a config option, but the red markers did not happen for my servers until I took this latest A3EAI update.  

When I turned it on, I was hoping the krypto helper/finder would surface the krypto, so it was either near field always present in the mouse scroll menu, or make it easier to physically aim and pick it up with the click action.  

I think I will just turn it off then.  Thank you!
